langlib/langlib-test/src/test/resources/test-src/stringlib_test.bal

```diff
@@ -35,14 +35,14 @@ function testIterator() returns string[] {
     string str = "Foo Bar";
 
     object {
-         public isolated function next() returns record {| string value; |}?;
+         public isolated function next() returns record {| string:Char value; |}?;
```

Comment:
Shall we also add a new test with string(not Char) also? It should work as we return Char right?

Reply:
Okay. I'll add.